cmd/hipsterfy/Hipsterfy/Server/Pages.hs

```diff
@@ -14,6 +13,8 @@ import Text.Blaze.Html5 hiding (body, contents, head, map)
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
 
+data PageError = FriendCodeError
```

Comment:
This type isn't very useful - it doesn't actually carry any information, isn't part of a sum, and if you _really_ wanted to use `Either`, you could use either `Either () r` or its isomorphic equivalent `Maybe r`.